its/src/test/java/com/sonar/it/csharp/IncrementalAnalysisTest.java

```diff
@@ -23,21 +23,20 @@
 import com.sonar.it.shared.TestUtils;
 import com.sonar.orchestrator.build.BuildResult;
 import com.sonar.orchestrator.build.ScannerForMSBuild;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.sonarqube.ws.Duplications;
-import org.sonarqube.ws.Issues;
-
-import java.io.BufferedWriter;
 import java.io.BufferedReader;
+import java.io.BufferedWriter;
 import java.io.File;
-import java.io.FileWriter;
 import java.io.FileReader;
+import java.io.FileWriter;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.List;
+import org.junit.Before;
```

Comment:
@csaba-sagi-sonarsource Can you verify that your IntelliJ settings matches https://github.com/SonarSource/sonar-developer-toolset ?

I've got quite some autoformatting changes in this file.